src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -2097,11 +2216,17 @@
           short-summary: Direction of the packet relative to the VM.
         - name: --protocol
           short-summary: Protocol to test.
+    examples:
+        - name: Run test-ip-flow verify to test logical connectivity from a VM to the specified destination IPv4 address and port.
+          text: az network watcher test-ip-flow --resource-group resourceGroupName --direction Outbound  --protocol TCP --local 10.0.0.4:* --remote 10.1.0.4:80 --vm vmName
 """
 
 helps['network watcher test-connectivity'] = """
     type: command
-    short-summary: Test if a direct TCP connection can be established between a Virtual Machine and a given endpoint.
+    short-summary: (Preview) Test if a connection can be established between a Virtual Machine and a given endpoint.
```

Comment:
Standard is to use `(PREVIEW)` (all caps) as the leader to make the preview status more obvious.

Reply:
Addressed.